### DISCUSSION TITLE
Change parameters of audio clips with LFOs

### DISCUSSION THREAD

**Discussion by wargoth** [SynthstromAudible/DelugeFirmware#755] [Ideas]
I sample external gear a lot and I use audio clips for that.  But currently with official firmware there is no way to change any parameters of audio clips with LFOs. Workaround is to create a kit with the audio and then all automation is available. This limitation doesn't make sense from performance point of view. It would be nice to close the feature gap between kits and audio clips.

**Comment by n3ptun3s** [comment]
That would be super handy.
I also do not understand why it is not implemented in the original firmware.
